hydroshare/urls.py

```diff
@@ -23,7 +24,6 @@
     # Change the admin prefix here to use an alternate URL for the
     # admin interface, which would be marginally more secure.
     url("^admin/", include(admin.site.urls)),
-    url(r'^o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
```

Comment:
This should not be removed. I guess after you bring your branch up to date with develop branch, this change should go away.